react/week2/newtodo-list-app/src/TodoList.js

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import Counter from "./Counter";
+import getToDos from "./getToDos";
+import ToDoListRow from "./ToDoListRow";
+
+const ToDoItems = getToDos();
+
+const TodoList = () => {
+  const [item, setItem] = React.useState(ToDoItems);
+  //Add new item
```

Comment:
I think it is a good practice to make comments like this for yourself when you are crunching through a problem, but remember to remove them afterwards. In this case I think that the name of the function, AddnewItem, says exactly the same as the comment above it, so there's no need to have the comment here.